components/director/internal/metrics/collector.go

```diff
@@ -9,7 +11,8 @@ import (
 
 // Config configures the behaviour of the metrics collector.
 type Config struct {
-	EnableClientIDInstrumentation bool `envconfig:"default=true,APP_METRICS_ENABLE_CLIENT_ID_INSTRUMENTATION"`
+	EnableClientIDInstrumentation bool     `envconfig:"default=true,APP_METRICS_ENABLE_CLIENT_ID_INSTRUMENTATION"`
+	CensoredFlows                 []string `envconfig:"APP_METRICS_CENSORED_FLOWS"`
```

Comment:
CensoredFlows is an array here but in the values.yaml file is a single string, shouldn't be model there as an array too? Or perhaps as a single value in the configuration?

Reply:
As far as I'm informed, the `envconfig` library can't handle `[]`. It expects to parse arrays represented as a single string composed of comma separated values (reference: [its docs](https://pkg.go.dev/github.com/vrischmann/envconfig)).